content/posts/charting-a-course/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Charting a Course for Open Source Innovation"
+date: 2024-06-12
+draft: false
+description: "UC Berkeley Joins Multi-Campus Effort Supported by $1.85 Million Grant from Alfred P. Sloan Foundation to Advance Open Source Initiatives Across UC System."
+tags: [ospo]
+displayInList: true
+author: ["Jarrod Millman <jarrod-millman>"]
+
+resources:
+  - name: featuredImage
+    src: "kickoff.png"
+    params:
+      description: "UC-wide Network of OSPOs kickoff meeting at UCSC (May 31, 2024)"
+      showOnTop: true
+---
+
+On May 31, 2024, I participated in the kick-off event for our recently announced UC-wide network of Open Source Program Offices (OSPOs), held at the Institute for Arts and Sciences in Santa Cruz.
+Despite almost a year of regular video conferences to plan this collaboration and develop the proposal that funds the formation of our network of OSPOs, this was the first time many of us had the opportunity to interact with each other in person.
+
+This event brought together team members from UC Berkeley, UC Davis, UC Los Angeles, UC Santa Barbara, UC Santa Cruz, and UC San Diego.
+Collectively, we represent a diverse range of departments and roles, including library services, research computing, data science initiatives, and academic departments focused on computer science and engineering.
```

Comment:
Collectively, we represent a diverse range of departments and roles, including library sciences, research computing, data science initiatives, and academic departments focused on computer science and engineering. _-> or maybe "research libraries" as a unit? Not sure, but just trying to capture the expertise of our library colleagues_